src/transformers/pipelines.py

```diff
@@ -1426,15 +1452,14 @@ def __call__(self, *args, **kwargs):
             ]
 
             for idx, label_idx in filtered_labels_idx:
+                start_ind, end_ind = offset_mapping[idx]
+                word_ref = sentence[start_ind:end_ind]
+                word = self.tokenizer.convert_ids_to_tokens([int(input_ids[idx])])[0]
+                is_subword = len(word_ref) != len(word)
```

Comment:
changed `is_subword` detection logic: by comparing length of token(##token) with the original text span mapping (Assuming subwordpieces get prefixed by something). 
Incase the user wants some other logic they can first get ungrouped entities add `is_subword:bool` field to entities and call `pipeline.group_entities` themselves.